pkg/webui/console/views/device-general-settings/index.js

```diff
@@ -100,7 +102,9 @@ const DeviceGeneralSettings = () => {
     const {
       ids: { dev_eui: devEui, join_eui: joinEui },
     } = device
-    await dispatch(attachPromise(unclaimDevice(appId, devId, devEui, joinEui)))
+    await dispatch(
+      attachPromise(unclaimDevice(appId, devId, hexToBase64(devEui), hexToBase64(joinEui))),
```

Comment:
Should we do this in the SDK instead ? We don't have to edit all of the callers to this format then.

Reply:
The thing is that the changes in https://github.com/TheThingsNetwork/lorawan-stack/pull/6325 will remove EUIs from this request.

Reply:
That's right, but if we want to support the use case in which we allow unclaiming if the device is not registered, I think we should keep the identifiers around. Otherwise you cannot know what EUIs to unclaim.

Reply:
I agree, if there's a requirement to convert to base64 then we should do that on SDK level so we keep handling of byte data consistent.

Reply:
Ok is that small enough to quickly add to this PR? Unclaiming is currently broken and I don't know where to look to update something in the SDK.